File: CS/WinSolution.Module/MyPerson.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;
using DevExpress.Persistent.Base;

namespace WinSolution.Module {
    [DefaultClassOptions]
    public class MyPerson : Person {
        public MyPerson(Session session) : base(session) { }
    }
}

File: VB/WinSolution.Module/MyPerson.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo
Imports DevExpress.Persistent.Base

Namespace WinSolution.Module
	<DefaultClassOptions> _
	Public Class MyPerson
		Inherits Person
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
	End Class
End Namespace
